### ISSUE TITLE
Property 'hex' does not exist on type 'ColorValue'.

### ISSUE THREAD

**Issue by egazarian** [mikbry/mui-color#6]
### Version
typescript: 4.1.2
mui-color: 2.0.0-beta.2
@mui/material: 5.0.4
react: 17.0.2
@emotion/react: 11.5.0
@emotion/styled: 11.3.0

### Issue

When trying to access to the hex value of a ColorValue inside of ColorPicker onChange event handler, typescript throw **Property 'hex' does not exist on type 'ColorValue'.**
Same issue for all the properties of ColorValue object.

### Screenshot
![Capture d’écran 2022-04-14 à 10 16 19](https://user-images.githubusercontent.com/59256619/163343617-6f79da58-6e28-4147-8106-4c13c8cac844.png)

### Workaround
```tsx
<ColorPicker
    onChange={(e) => {
        const color = e as ColorObject;
        console.log(color.hex);
    }}
/>
```

Should the onChange handler argument type be ColorObject instead of ColorValue ?

**Comment by Briimbo** [comment]
I don't think this is a bug. You can also enter text in the color picker to get a color. The argument type used to be `Color` though instead of `ColorValue`
![grafik](https://user-images.githubusercontent.com/44522777/165981610-bfdb7b2f-0409-456f-8df8-ad24d4af2ffa.png)

**Comment by egazarian** [comment]
Ok, I got it. And yes, it would be better to have an argument of type `Color`, instead of `ColorValue`. In this way we should be able to get the color as any format.
